src/performance/dashboard/src/components/resultsCard/__tests__/ResultsCard_Blank.test.js

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+* Copyright (c) 2019 IBM Corporation and others.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v2.0
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     IBM Corporation - initial API and implementation
+*******************************************************************************/
+
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import ResultsCard_Blank from '../ResultsCard_Blank';
+
+// initialize component props
```

Comment:
Personal preference but I wouldn't have this comment, and probably not 'run tests' either. (This comment is basically the same as the variable name)?